includes/wf_crm_admin_form.inc

```diff
@@ -677,7 +677,24 @@ class wf_crm_admin_form {
       '#title' => t('Block Form Access when Event(s) are Full/Ended'),
       '#default_value' => (bool) wf_crm_aval($this->data, 'reg_options:block_form'),
     );
+    $this->form['participant']['reg_options']['disable_unregister'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Disable unregistering participants from unselected events.'),
+      '#default_value' => (bool) wf_crm_aval($this->data, 'reg_options:disable_unregister'),
+    );
+    $this->form['participant']['reg_options']['allow_url_load'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Allow events to be autoloaded from URL'),
+      '#default_value' => (bool) wf_crm_aval($this->data, 'reg_options:allow_url_load'),
+      '#description' => t('Example for "Register all":') .
+      '<br /><code>' . url("node/{$this->node->nid}", array('absolute' => TRUE, 'query' => array('event1' => ''))) . '{event1.event_id},{event2.event_id}</code></p>'.
+      t('Example for "Register separately":') .
+      '<br /><code>' . url("node/{$this->node->nid}", array('absolute' => TRUE, 'query' => array('c1event1' => ''))) . 
+      '{event1.event_id},{event2.event_id}&amp;c2event1={event3.event_id}</code></p>',
```

Comment:
I think this help text ought to be in the help popup instead, to de-clutter the form.